sdk/loadtestservice/azure-developer-loadtest/README.md

```diff
@@ -0,0 +1,109 @@
+
+# Azure Developer LoadTesting client library for Python
+<!-- write necessary description of service -->
```

Comment:
Please write some overview about the service to override these words

Reply:
Added in latest commit